pkg/azurestore/azureservice.go

```diff
@@ -187,6 +191,31 @@ func (blockBlob *BlockBlob) Download(ctx context.Context) (io.ReadCloser, error)
 	return resp.Body, nil
 }
 
+// Serve content respecting range header
+func (blockBlob *BlockBlob) ServeContent(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
+	var downloadOptions, err = parseHTTPRange(r)
+	if err != nil {
+		return err
+	}
+	resp, err := blockBlob.BlobClient.DownloadStream(ctx, downloadOptions)
+	if err != nil {
+		return err
+	}
+
+	statusCode := http.StatusOK
+	if resp.ContentRange != nil {
+		// Use 206 Partial Content for range requests
+		statusCode = http.StatusPartialContent
+	} else if resp.ContentLength != nil && *resp.ContentLength == 0 {
+		statusCode = http.StatusNoContent
+	}
+	w.WriteHeader(statusCode)
```

Comment:
Can we forward response headers like Content-Type from Azure's response? Similar to how its done in the s3store: https://github.com/tus/tusd/blob/a50bc426ddd52a484dc476419c5bd77397ea05e6/pkg/s3store/serve_content.go#L95-L128